addon/-private/engine-instance-ext.js

```diff
@@ -230,21 +234,36 @@ EngineInstance.reopen({
     });
   },
 
+  getExternalRoute(routeName) {
```

Comment:
I'll go ahead and prefix this, since the RFC for this is still in flight, we should probably err on the side of caution with the APIs we expose.